std_msgs/String missionType
std_msgs/String random_password
std_msgs/String recipient
std_msgs/String recipient_location
std_msgs/String sender
std_msgs/String sender_location
std_msgs/Int8 priority
